.test-defs/infrastructure-test.yaml

```diff
@@ -0,0 +1,20 @@
+kind: TestDefinition
+metadata:
+  name: infrastructure-test
+spec:
+  owner: DL_5bac5a9aecb2116334000005@exchange.sap.corp
+  description: Integration test for infrastructure creation and deletion
+  activeDeadlineSeconds: 1800
+
+  command: [bash, -c]
+  args:
+    - >-
+      go test -timeout=0 -mod=vendor ./test/integration/infrastructure
```

Comment:
You could also add `integration-test-infra` target in the Makefile to make it consistent with other provider extensions.